README.md

```diff
@@ -23,7 +23,7 @@ This Kubernetes Operator is meant to be deployed in your Kubernetes cluster(s) a
 
 First you need to deploy Tower Operator into your cluster:
 
-    kubectl apply -f https://raw.githubusercontent.com/geerlingguy/tower-operator/master/deploy/tower-operator.yaml
```

Comment:
The one thing I like about this particular line is the fact that you don't have to have this repo cloned locally in order to deploy to your cluster. You can copy and paste this line, and the operator is there. Also, there's a playbook that builds the `tower-operator.yaml` file that would be removed if we drop that file. (And some related notes in the README).

Reply:
Actually as I'm coming up to speed on kustomize, I realize you can do something similar without having to pull down the whole repo. So we could do something like the following (for my specific branch so you can see this working for now):

`kubectl apply -k github.com/tylerauerbeck/tower-operator//deploy?ref=add-kustomize`

So you're able to execute this remotely and can even specify the branch/release you want to work with. 

As for the playbook that builds the `tower-operator.yaml` .. you don't actually need to build that "all-in-one" file anymore. Kustomize brings all of the individual files together so that doesn't actually need assembled anymore (unless there's a specific reason that you're doing that for that I'm just not aware of).

Reply:
@geerlingguy Just wanted to revisit this as I just noticed the issue had gone stale. With the change I just mentioned above, it looks like it should give the same experience that you're looking for.

That being said, if you're looking to go in a different direction, that's fine as well. Just wanted to address this before the issue itself gets closed.